public/playerDecisionController.js

```diff
@@ -0,0 +1,130 @@
+"use strict";
+
+export default class playerDecisionController {
+  constructor(players, generatorCardObject) {
+    if (!Array.isArray(players)) {
+      throw new Error(
+        `You must provider an array with the players to the playerDecisionController constructor`
+      );
+    }
+
+    this.players = players;
+    this.amountOfPlayers = this.players.length;
+    this.generatorCardObject = generatorCardObject;
+    this.currentPlayerTurn = this.definePlayerTurn();
+
+    const controlsContainerEle = document.querySelector("#game");
+    this.controlsContainerEle = controlsContainerEle;
```

Comment:
Same as before, you are defining a variable before this statement just to assign it next, so makes no sens to define the initial variable